lib/commands/general.js

```diff
@@ -11,6 +11,12 @@ commands.getPageSource = async function () {
   return await this.uiautomator2.jwproxy.command('/source', 'GET', {});
 };
 
+commands.getClipboard = async function () {
+  return (await this.adb.getApiLevel() < 29)
```

Comment:
I noticed https://github.com/appium/appium-adb/pull/473/files#r347388767 was needed since the setting app already returned data as base64 encoded string.

After the change, this logic worked on android 10 👍